src/ripple/app/main/Application.cpp

```diff
@@ -844,7 +854,9 @@ class ApplicationImp
 
         mValidations->flush ();
 
-        m_overlay->saveValidatorKeyManifests (getWalletDB ());
+        validators_->stop ();
+
+        manifestCache_->save (getWalletDB ());
```

Comment:
`wallet.db` was the "we'll put everything in there" database. Not only is it poorly named (wallet what now?) but it contains tables that haven't been used since around the time I joined what... 3 years ago?

The **only** important bit of information in there is the node identity (see https://github.com/ripple/rippled/blob/develop/src/ripple/app/main/NodeIdentity.cpp#L56) and storing that in `wallet.db` is actually a bad idea and makes the file into a kind of hidden "config" file: wipe this file and your server gets a new unique identity.

Also note that this `wallet.db` is an application-level database (https://github.com/ripple/rippled/blob/develop/src/ripple/app/main/Application.cpp#L363) and for what reason? 

The only other use is for manifests, and I think ought to consider having them use their own, separate, database.